pkg/api/v2/join.go

```diff
@@ -177,12 +178,6 @@ func (a *API) Join(ctx context.Context, req JoinRequest) (*JoinResponse, int, er
 		if err := a.Snap.AddCertificateRequestToken(fmt.Sprintf("%s-proxy", req.ClusterToken)); err != nil {
 			return nil, http.StatusInternalServerError, fmt.Errorf("failed adding certificate request token for kube-proxy: %w", err)
 		}
-
```

Comment:
We have to continue supporting the old API since someone may want to join a 1.24 node to a 1.25 control plane.

We may want to create a v3.

Reply:
I'm fine with keeping the list here, it was removed only because it's not necessary anymore. Once the worker upgrades, the proxy will automatically get all the endpoints.

> someone may want to join a 1.24 node to a 1.25 control plane

This would work, but is not really supported and could cause problems.

Reply:
I can revert the change

Reply:
We can create a v3 version of the join command.